resources/assets/js/components/forms/createTaskForm.vue

```diff
@@ -285,6 +291,23 @@ export default {
       this.assignedTo = ''
       this.dueOnDate = null
       this.relatedTo = ''
+    },
+    changeDueDate(value) {
+      let cycleSelected = this.cycles.find(cycle => cycle.id === value)
+      let todayDate = new Date()
+      let startDate = new Date(cycleSelected.start_date)
+      startDate.setDate(startDate.getDate())
+      let endDate = new Date(cycleSelected.end_date)
+      endDate.setDate(endDate.getDate() + 1)
+
+      if(todayDate.getTime() < startDate.getTime()){
+        this.disabledDates.to = new Date(startDate)
+      }else {
+        this.disabledDates.to = new Date(todayDate)
```

Comment:
today date need to be one day less also, currently in a cycle (1Mar - 30Mar), today (28) not selectable. Please check